src/main/java/org/spongepowered/api/text/selector/SelectorBuilder.java

```diff
@@ -36,7 +35,8 @@
     /**
      * Sets the type of this selector.
      *
-     * @param type The type to set
+     * @param type
+     *            The type to set
      * @return This selector builder
      */
     SelectorBuilder selectorType(SelectorType type);
```

Comment:
This is already in https://github.com/SpongePowered/SpongeAPI/commit/3ecf15584fdf5515f5bd065f708fc7177c6711de